aten/src/ATen/cpu/vec/vec256/vec256_qint.h

```diff
@@ -1064,8 +1124,20 @@ struct Vectorized<c10::quint8> : public VectorizedQuantizedConverter<
             std::array<Vectorized<c10::qint32>, 4>,
             32>(ptr) {}
 
-  static Vectorized<c10::quint8> loadu(const void* ptr) {
-    return Vectorized<c10::quint8>(ptr);
+  static Vectorized<c10::quint8> loadu(const void* ptr, int64_t count = size()) {
```

Comment:
Can we please follow the same pattern for both vec256_quint and vec512_quint 
(I understand that size() is constexpr and compiler should be able to optimize out if when invoked with default parameters, but older compilers might not be as smart)
```suggestion
  return Vectorized<c10::quint8>(ptr)l
  }
  static Vectorized<c10::quint8> loadu(const void* ptr, int64_t count) {
```

Reply:
Thanks. It's fixed.